tools/ci-scripts/codegen-diff/semver-checks.py

```diff
@@ -28,7 +28,7 @@ def main(skip_generation=False):
 
     if not skip_generation:
         checkout_commit_and_generate(head_commit_sha, CURRENT_BRANCH, targets=['aws:sdk'])
-        checkout_commit_and_generate(base_commit_sha, BASE_BRANCH, targets=['aws:sdk'], preserve_aws_sdk_build=True)
+        checkout_commit_and_generate(base_commit_sha, BASE_BRANCH, targets=['aws:sdk'], preserve_aws_sdk_build=False)
```

Comment:
intentional?

Reply:
Reverted in https://github.com/smithy-lang/smithy-rs/pull/3936/commits/5f0c533be2a6ac951b9084b6853a5bea6aa2baaa